app.js

```diff
@@ -13,6 +13,52 @@ function hasFilter(name, filters) {
   return false;
 }
 
+function searchToUrl(search) {
+  if (!search.toString()) {
+    return "";
+  }
+  return "?" + search.toString();
+}
+
+function initialSetStateByUrl() {
```

Comment:
Is this function really needed ? If an user navigates to devtools-bug-finder with a query string, there's no need set the query string once again with the parsed value.

What I would do is replace the call to this function in `init` with: 
```
if (!location.search) {
  return setDefaultState();
}
```